.github/workflows/ci.yaml

```diff
@@ -33,7 +33,10 @@ jobs:
         pip freeze | safety check
     - name: Lint with Flake8
       run: |
-        flake8
+        # stop the build if there are Python syntax errors or undefined names
```

Comment:
Hmmm, I'll need convincing that turning off flake8 checks for CI is a good thing....

Reply:
This was primarily to get a build out for testing - note that linting is not bypassed entirely, just limited to functional checks (syntax errors, undefined names), but still running and reporting on the more stylistic guidelines - just not cancelling the build. It can certainly be tuned differently than I have here, I just didn't want to hassle with e.g. unused imports, whitespace, and line length violations while in development. Admittedly, this is debatable - it may be better to enforce the more stylistic checks sooner rather than later so they don't rack up. 

We could also set up a stricter CI for e.g. pushes to a main branch, but may want to make sure we have some differentiation in the resulting image tags (e.g. 'dev' and 'prod', or something of the like) - I haven't looked too closely at the 'publish' section of the github actions tho.

TL,DR; I'm fine to return to full force flaking (minus line length limits) if that's the call.